ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -809,6 +809,24 @@ tests:
         AGENT_USE_APPLIANCE_MODEL=true
     workflow: agent-e2e-compact-ipv4
   timeout: 6h0m0s
+- as: e2e-agent-compact-ipv4-appliance-diskimage
+  cluster: build05
+  optional: true
+  run_if_changed: /agent/
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: equinix-ocp-metal
+    dependencies:
+      APPLIANCE_IMAGE: agent-preinstall-image-builder
+    env:
+      AGENT_E2E_TEST_BOOT_MODE: DISKIMAGE
```

Comment:
Just noticed this one, looks like a type

Reply:
> Just noticed this one, looks like a type

Which typo?

Reply:
The `AGENT_E2E_TEST_BOOT_MODE` is defined both has env var and `DEVSCRIPTS_CONFIG` config var. I was expecting just the latter?

Reply:
We need both since we use it as an indication for whether to add APPLIANCE_IMAGE env var to config_root.sh.
See: https://github.com/openshift/release/pull/42532
We use this method in also in all appliance jobs: https://github.com/openshift/release/blob/a2283ea2799d9f5392cae93e07f8aa1443473778/ci-operator/config/openshift/appliance/openshift-appliance-master.yaml#L77

Reply:
Ah, right, forgot about that, thanks for clarifying